backend/src/main/scala/cromwell/backend/io/WorkflowPaths.scala

```diff
@@ -3,15 +3,15 @@ package cromwell.backend.io
 import java.nio.file.{Path, Paths}
 
 import com.typesafe.config.Config
-import cromwell.backend.{BackendJobDescriptorKey, BackendWorkflowDescriptor}
+import cromwell.backend.BackendWorkflowDescriptor
 
 object WorkflowPaths{
   val DockerRoot = Paths.get("/root")
 }
 
 class WorkflowPaths(workflowDescriptor: BackendWorkflowDescriptor, config: Config) {
   // TODO will need update when configuration is rearranged
```

Comment:
Yes I think I wrote this comment, it was at the time when @mcovarr was re-arranging the config and the value for the base directory attribute (which is now "root") wasn't fully defined or whatever. It can be removed.